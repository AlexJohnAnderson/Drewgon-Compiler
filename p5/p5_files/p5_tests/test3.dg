bool p;
int foo(int a){

if(true && ){
a = p;
}

return p;
}

