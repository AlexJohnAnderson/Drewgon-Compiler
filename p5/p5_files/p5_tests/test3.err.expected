FATAL [4,9]-[4,10]: Illegal character &
FATAL [4,10]-[4,11]: Illegal character &
FATAL [5,1]-[5,6]: Invalid assignment operation
FATAL [8,8]-[8,9]: Bad return value
Type Analysis Failed
